downstairs/src/repair.rs

```diff
@@ -77,14 +77,7 @@ pub async fn repair_main(
             .start();
     let local_addr = server.local_addr();
 
-    tokio::spawn(async move {
-        /*
-         * Wait for the server to stop.  Note that there's not any code to
-         * shut down this server, so we should never get past this
-         * point.
-         */
-        server.await
-    });
+    tokio::spawn(server);
```

Comment:
I think we will be back here shortly for all the cancel safe stuff.

Reply:
:grimacing: